finetune/base_models/bert/featurizer.py

```diff
@@ -119,3 +121,24 @@ def bert_featurizer(
             output_state = {k: tf.stop_gradient(v) for k, v in output_state.items()}
 
         return output_state
+
+def bert_featurizer_with_pos(
+    X,
+    context,
+    encoder,
+    config,
+    train=False,
+    reuse=None,
+    **kwargs
+):
+    featurizer_state = bert_featurizer(X, encoder, config, train=train, reuse=reuse, **kwargs)
+    with tf.variable_scope("model/featurizer"):
+        embed_position(context, featurizer_state, config, train)
+        add_context_embed(featurizer_state)
+        hidden = featurizer_state['sequence_features']
+        w0, w = smooth_pos_attn(hidden, config, featurizer_state['lengths'])
+        text_embed = hidden[:, :, :config.n_embed]
+        seq_feats = tf.matmul(w, text_embed)
```

Comment:
I think we should use a more flexible / general arch. for our pretraining experiments than we used for our finetuning experiments. IMO a reasonable baseline would be:

- Concat position embeddings to contextual embeddings output by base roberta
- Add 1-2 additional transformer layers that operate on the concatted vector
- Maybe worry about how we handle initialization (Can we make sure the new layers are dominated by residuals at the start so our hidden --> n_vocab projection for the MLM loss is still valid)?